### PR TITLE
Select estimated fee UI

### LINKED ISSUE #83: Add estimated fee in create vtt form
Allow selected estimated fee in create vtt form

### PR DESCRIPTION
**Pull request by gabaldon**
close #83

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -14,7 +14,6 @@ dev_dependencies:
 dependencies:
   flutter:
     sdk: flutter
-
```
Comment (Tommytrg) [inline pubspec.yaml:17]: We can leave it as it was

File: macos/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -159,7 +159,6 @@
 				9F701DC45E7C54B384D50870 /* Pods-Runner.release.xcconfig */,
 				17638EA08E7446DFA218BF3A /* Pods-Runner.profile.xcconfig */,
 			);
-			name = Pods;
```
Comment (Tommytrg) [inline macos/Runner.xcodeproj/project.pbxproj:162]: Is this file part of the build?

File: lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/02_select_miner_fee.dart [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+  }
+
+  NavAction next() {
+    return NavAction(
+      label: 'Continue',
+      action: nextAction,
+    );
+  }
+
+  Widget _buildFeeOptionButton(EstimatedFeeOptions label, String value) {
```
Comment (Tommytrg) [inline lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/02_select_miner_fee.dart:218]: When there is an error, all borders should be red. Right now, one of them is black.

**Review by Tommytrg — CHANGES_REQUESTED** [review]

File: lib/bloc/explorer/api_explorer.dart [hunk 4]
```diff
@@ -27,6 +31,7 @@ enum ExplorerQuery {
 class ApiExplorer {
   late ExplorerClient client;
   late Status status;
+  TransactionCache cache = TransactionCache();
```
Comment (parodyBit) [inline lib/bloc/explorer/api_explorer.dart:34]: `TransactionCache` is no longer used and we should use the `WalletStorage` from the database.

File: lib/bloc/transactions/value_transfer/vtt_create/vtt_create_bloc.dart [hunk 5]
```diff
@@ -307,7 +342,7 @@ class VTTCreateBloc extends Bloc<VTTCreateEvent, VTTCreateState> {
 
   /// add a [ValueTransferOutput] to the [VTTransaction].
   void _addValueTransferOutputEvent(
-      AddValueTransferOutputEvent event, Emitter<VTTCreateState> emit) {
+      AddValueTransferOutputEvent event, Emitter<VTTCreateState> emit) async {
```
Comment (Tommytrg) [inline lib/bloc/transactions/value_transfer/vtt_create/vtt_create_bloc.dart:345]: Does this function need to be async?
```suggestion
      AddValueTransferOutputEvent event, Emitter<VTTCreateState> emit) {
```

**Pull request merged**